src/DIRAC/Core/Utilities/File.py

```diff
@@ -139,7 +139,7 @@ def checkGuid(guid):
     return False
 
 
-def getSize(fileName):
+def getSize(fileName: str) -> int:
```

Comment:
FYI: This should be `os.PathLike` so `pathlib` also works:
```suggestion
def getSize(fileName: os.PathLike) -> int:
```

Reply:
thanks, updated